src/pathToVictory/services/officeMatch.service.ts

```diff
@@ -460,13 +564,26 @@ export class OfficeMatchService {
 
       this.logger.debug(`searching for ${search}`)
 
-      let searchValues = await this.votersService.querySearchColumn(
+      const year =
+        electionDate && electionDate.length >= 4
+          ? Number(electionDate.slice(0, 4))
+          : undefined
+      if (!year) {
```

Comment:
Oh come on now.. we're just duplicating this across the codebase?

Use `date-fns` please